src/pages/home/report/ReportActionsList.js

```diff
@@ -192,6 +192,9 @@ class ReportActionsList extends React.Component {
                         this.setState({
                             skeletonViewHeight: event.nativeEvent.layout.height,
                         });
+                        if (ReportScrollManager && ReportScrollManager.layoutChange) {
```

Comment:
Add a no-op to `ReportScrollManager/index.js` like so 
```
function scrollToLastIndex() {}
```
so we don't have to check if ReportScrollManager.layoutChange exists.
Also, is there a case when ReportScrollManager would be null? Do we need this if condition?

Reply:
updated, you are correct wasn't sure it would be expected standart